engine/runtime-instrument-common/src/main/java/org/yaml/snakeyaml/introspector/MethodProperty.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2008, SnakeYAML
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.yaml.snakeyaml.introspector;
+
+// remove once https://github.com/snakeyaml/snakeyaml/pull/12 gets integrated
```

Comment:
Remove this file once https://github.com/snakeyaml/snakeyaml/pull/12 gets integrated.

Reply:
E.g. when upgrading to [SnakeYaml 2.3](https://bitbucket.org/snakeyaml/snakeyaml/wiki/Changes) to be released in ~August. Tracked as 
- #9113